<ui version="4.0" >
 <class>AddProjectDialog</class>
 <widget class="QDialog" name="AddProjectDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add Project</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>&amp;Name:</string>
       </property>
       <property name="buddy" >
        <cstring>nameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2" >
      <widget class="QLineEdit" name="nameEdit" >
       <property name="toolTip" >
        <string>Enter the name of the project</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Project&amp;file:</string>
       </property>
       <property name="buddy" >
        <cstring>filenameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="filenameEdit" >
       <property name="toolTip" >
        <string>Enter the name of the project file</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QPushButton" name="fileButton" >
       <property name="toolTip" >
        <string>Select the project file via a file selection dialog</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>&amp;Description:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="buddy" >
        <cstring>descriptionEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2" >
      <widget class="QTextEdit" name="descriptionEdit" >
       <property name="toolTip" >
        <string>Enter a short description for the project</string>
       </property>
       <property name="tabChangesFocus" >
        <bool>true</bool>
       </property>
       <property name="acceptRichText" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3" >
      <widget class="QCheckBox" name="masterCheckBox" >
       <property name="toolTip" >
        <string>Select to make this project the master project</string>
       </property>
       <property name="text" >
        <string>Is &amp;master project</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>filenameEdit</tabstop>
  <tabstop>fileButton</tabstop>
  <tabstop>descriptionEdit</tabstop>
  <tabstop>masterCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
